src/components/GATT Database.xml

```diff
@@ -176,7 +176,7 @@
 							<Descriptor uuid="2905" name="Aggregate Format" handle="175" permissions="1">A6 00 A9 00 AC 00</Descriptor>
 						</Descriptors>
 					</Characteristic>
-					<Characteristic uuid="B011" name="Value V17" handle="176" properties="66" permissions="3">12</Characteristic>
+					<Characteristic uuid="B011" name="Value V17" handle="176" properties="2" permissions="3">12</Characteristic>

```

Comment:
Please add a comment that this is a deviation from the reference data base that is needed because Zephyr does not support data signing and EDTT does not support an ICS to handle this more elegantly.